server/src/shared/infra/db/entities/tags/tag.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Collection, Entity, ManyToMany, Property } from '@mikro-orm/core'
+import { BaseEntity } from '../base.entity'
+import { ClubEntity } from '../club.entity'
+import { EventEntity } from '../event.entity'
+
+@Entity()
+export class TagEntity extends BaseEntity {
+  @Property()
+  name!: string
+  @ManyToMany({ entity: () => ClubEntity, inversedBy: "tags" })
```

Comment:
```suggestion

  @ManyToMany({ entity: () => ClubEntity, inversedBy: "tags" })
```

Reply:
@KTong821 can you accept this suggestion?